versioned_docs/version-3.0.0-beta/quick-start/upgrade-plan.md

```diff
@@ -37,13 +37,16 @@ Users can upgrade or downsize their team in Spheron's Pro membership model to fi
 
 #### PRO Package
 
-Currently, the <b>Pro</b> subscription plan costs $18 per month. Covering in features like
+Currently, the <b>Pro</b> subscription plan costs $10 per month per member. Covering in features like
```

Comment:
add these in the starter pack as well
- <b>10 ENS Domain Limit</b>
- <b>10 HNS Domain Limit</b>